charts/atlantis/templates/statefulset.yaml

```diff
@@ -541,10 +544,19 @@ spec:
             mountPath: /home/atlantis/.netrc
             subPath: netrc
           {{- end }}
-          {{- if or .Values.aws.credentials .Values.aws.config .Values.awsSecretName }}
+          {{- if .Values.awsSecretName }}
           - name: aws-volume
             readOnly: true
             mountPath: {{ .Values.aws.directory | default "/home/atlantis/.aws" }}
```

Comment:
Please add this to the `values.yaml` example so it's not hidden.

Reply:
I'm not sure what you're asking for here.   Just to indicate in the values.yaml that config and credentials now mount as individual files instead of a directory?

Reply:
I was saying to add `.Values.aws.directory` to the default values because I thought it was a new config, but looking further I see that it is not. Sorry for that.

Please bump the chart minor version so we can merge this!